packages/storybook/src/nimble/table/table.stories.ts

```diff
@@ -28,11 +28,23 @@ const metadata: Meta<BaseTableArgs> = {
     parameters: {
         actions: {
             handles: [
+                /*
```

Comment:
Shot in the dark, but I recall an issue with the Actions add-on where it can get broken / slow trying to print large / complex event data. I think I recall fixing it by limiting the depth of the event data serialization. I can help search for that if you think that might be the issue and can't discover how to configure it on your own.

Reply:
Yep, that's what I was recalling. Too bad it doesn't help 😔